doc/faq.rst

```diff
@@ -157,6 +157,32 @@ and CDAT have some great domain specific functionality, and we would love to
 have support for converting their native objects to and from xarray (see
 :issue:`37` and :issue:`133`)
 
+
+What other projects leverage xarray?
+------------------------------------
+
+Here are several existing libraries that build functionality upon xarray.
+They may be useful points of reference for your work:
+
+- `aospy <https://aospy.readthedocs.io>`_: Automated analysis and management of gridded climate data.
+- `eofs <https://ajdawson.github.io/eofs/>`_: EOF analysis in Python.
+- `MPAS-Analysis <http://mpas-analysis.readthedocs.io>`_: Analysis for simulations produced with Model for Prediction Across Scales (MPAS) components and the Accelerated Climate Model for Energy (ACME)
+- `psyplot <https://https://psyplot.readthedocs.io>`_: Interactive data visualization with python.
+- `PyGDX <https://pygdx.readthedocs.io/en/latest/>`_: Python 3 package for
+  accessing data stored in GAMS Data eXchange (GDX) files. Also uses a custom
+  subclass.
+- `salem <https://salem.readthedocs.io>`_: Adds geolocalised subsetting,
```

Comment:
was: "Adds geolocalised subsetting, masking, and plotting operations to xarray's data structures via accessors."

(the second line is gone in your PR)